.github/workflows/CI.yml

```diff
@@ -200,11 +200,18 @@ jobs:
           echo "::set-output name=KEPTN_SPEC_VERSION::${KEPTN_SPEC_VERSION}"
       - name: Get current date and time
         id: get_datetime
+        env:
+          GO_VERSION: ${{ env.GO_VERSION }}
         run: |
           echo "::set-output name=DATE::$(date +'%Y%m%d')"
           echo "::set-output name=TIME::$(date +'%H%M')"
           echo "::set-output name=DATETIME::$(date +'%Y%m%d')$(date +'%H%M')"
 
+      - name: Get GO-Version
+        id: get_go_version
+        run: |
+          echo "::set-output name=GO_VERSION::$GO_VERSION"
```

Comment:
I don't think you need this the go version is defined globally in the workflow at the top